poly-commitment/src/commitment.rs

```diff
@@ -866,8 +866,7 @@ impl<G: CommitmentCurve> SRS<G> {
         }
 
         // verify the equation
-        let scalars: Vec<_> = scalars.iter().map(|x| x.into_repr()).collect();
-        VariableBaseMSM::multi_scalar_mul(&points, &scalars) == G::Projective::zero()
+        G::Group::msm_unchecked(&points, scalars.as_slice()) == G::Group::zero()
```

Comment:
I'm wondering if this should be a checked `msm` here

Reply:
https://github.com/o1-labs/proof-systems/pull/979 actually fixes the code so that it's always a checked `msm`, so we don't have to care too much in this PR and changing everything to `msm_unchecked` is good enough at this point

Reply:
I thought about it, and decided to keep close to the original semantics which was unchecked.